BUILD.bazel

```diff
@@ -82,3 +112,13 @@ container_push(
     tag = "latest",
     visibility = ["//visibility:public"],
 )
+
+container_push(
+    name = "push_to_dockerhub-arm64",
+    format = "Docker",
+    image = ":bazel-remote-image-arm64",
+    registry = "index.docker.io",
+    repository = "buchgr/bazel-remote-cache-arm64",
```

Comment:
Let met know what you think about this naming. If we can agree on a repo name, since you have push-rights to this repo on `github`, I assume you're also maintaining the `hub` repo. Can you please test if this works?

Reply:
I don't have access to dockerhub, unfortunately. Let's skip this part for now.

Reply:
OK, we can skip pushing. We need to understand if the correct image is being build. Let's continue this discussion in [this](https://github.com/buchgr/bazel-remote/pull/215#discussion_r386026640) thread. I'll resolve this conversation once I remove the code for pushing from the `bazel` file.

Reply:
Do you still want to leave in the rule for pushing to `buchgr/bazel-remote-cache-arm64`? I would leave it, once you have access you can invoke it. If not, just don't use it, I guess.

Reply:
Let's remove the rule for now. We can add it back later if it looks like we can commit to updating the docker arm64 images.